docs/submit/osg-flock.md

```diff
@@ -1,97 +1,170 @@
+* Obtain root access to the host
+* Prepare [the required Yum repositories](/common/yum.md)
+* Install [CA certificates](/common/ca.md)
 
-The requirements are:
 
-* A public IP address, forward and reverse DNS.
-* Ability to open a few firewall incoming ports to the WAN.
-* HTCondor has to authenticate via pool GSI (preffered) or password. For GSI, the submit host
-   has to have a host certificate.
-* Reporting to the OSG accounting system has to be enabled. This can
-   be accomplished by installing and configuring the *gratia-probe-condor* and *gratia-probe-glideinwms* RPMs.
-* Submitted jobs should have the *+ProjectName* attribute specified with
-   a valid registered project name.
+Initial Steps
+-------------
 
+### Register your submit host in OSG Topology
+To be part of OSG, your submit host should be registered with the OSG.
+You will need information like the hostname, and the administrative and security contacts.
+Follow the [general registration instructions](/docs/common/registration#new-resources).
+The service type is `Submit Node`.
+
+
+### Choose your authentication method
+There are two options for authentication of your submit host to the OSG: GSI and pool password.
+Of these, GSI is the recommended method, but it will require obtaining a host certificate for your submit host.
+See our [documentation](/security/host-certs.md) for instructions on how to request and install host certificates.
+
+If you are unable to obtain a host certificate, use the "pool password" authentication method, described below.
 
-Required Packages
------------------
 
-Enable the [OSG Yum repository](/common/yum/).
+### Request to be allowed to flock to OSG
+OSG staff will need to add your submit host to the list of hosts that flocked jobs are accepted from.
+Send email to <help@opensciencegrid.org> with the hostname of your submit host and request to be added to the list.
 
-Install the packages required:
+
+Installing Required Software
+----------------------------
+Flocking requires HTCondor software as well as software for reporting to the OSG accounting system.
+OSG provides a convenience RPM that installs all required packages.
+To install, run:
 
 ```console
 root@host # yum install osg-flock
 ```
 
-Gratia Probe Configuration
---------------------------
+
+Configuring Reporting via Gratia
+--------------------------------
+Reporting to the OSG accounting system is done using the _Gratia_ service, which consists of multiple _probes_.
+HTCondor uses the "condor" probe, which is configured in `/etc/gratia/condor/ProbeConfig`;
+we provide a recommended config for flocking.
 
 1. Copy over the recommended probe configuration:
 
         :::console
         root@host # cp /etc/gratia/condor/ProbeConfig-flocking /etc/gratia/condor/ProbeConfig
 
-1. Fill in the values for `ProbeName` and `SiteName` with the hostname and Topology resource name, respectively. For example:
+1. Fill in the value for `ProbeName` with the hostname of your submit host.
```

Comment:
I would add something in between indicating the format so they know what things they should change:

```
ProbeName="condor:<HOSTNAME>"
````